kaolin/datasets/modelnet.py

```diff
@@ -12,63 +12,62 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Callable, Iterable, Optional, Union, List
+from typing import Iterable, Optional
 
 import torch
 import os
 from glob import glob
-from tqdm import tqdm
 
 from kaolin.rep.TriangleMesh import TriangleMesh
-from kaolin.transforms import transforms as tfs
 
 
 class ModelNet(object):
```

Comment:
It's not using KaolinDataset as a base class

Reply:
Done